packages/realm/src/app-services/User.ts

```diff
@@ -163,14 +167,21 @@ export class User<
    * If this value has not been configured, it will be empty.
    */
   get customData(): CustomDataType {
-    throw new Error("Not yet implemented");
+    const result = this.internal.customData;
```

Comment:
maybe it would be worth having a cached type for this as well?

Reply:
It gets updated when the access token gets refreshed, so I think it's better to leave it un-memoized for now.
Also, it's not too expensive compared to how little it'll get called and end-users can always work around by caching it on the calling side if they experience performance issues.